pages/1.11/installing/ent/custom/configuration/configuration-parameters/index.md

```diff
@@ -57,6 +57,12 @@ This topic provides configuration parameters available for [DC/OS Enterprise](ht
 | [dcos_l4lb_enable_ipv6](#dcos_l4lb_enable_ipv6)        | A boolean that indicates if layer 4 load balancing is available for IPv6 networks. |
 |[dcos_ucr_default_bridge_subnet](#dcos-ucr-default-bridge-subnet) |IPv4 subnet allocated to the `mesos-bridge` CNI network for UCR bridge-mode networking. | 
 
+# Storage 
+
+| Parameter                    | Description                                                                                                                                                       |
+|------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [feature_dcos_storage_enabled](#feature-dcos-storage-enabled-enterprise)           | A flag that enables [CSI support](https://github.com/container-storage-interface/spec) on the Mesos agents. By default set to `false`. This feature flag needs to be turned on to use the [Data Storage Service (DSS)](/services/beta-storage)|
```

Comment:
I'd say: `A flag, if set, will enable advanced storage features in DC/OS, including Mesos CSI support, pre-loaded device plugins, etc. This feature flag needs to be turned on to use the [DC/OS Storage Service (DSS)](/services/beta-storage)`